adr/0002-API-params-naming.md

```diff
@@ -0,0 +1,27 @@
+# 1. Record architecture decisions
+
+Date: 2020-02-06
+
+## Status
+
+Accepted
+
+## Context
+
+The naming of the params used in the API needs be consistent.
+
+For example:
+nomis_offender_id, prison_number and nomis_prison_number refer to the same data.
+
+## Decision
+Use param names similar to those used on [DPS APIs](https://gateway.prod.nomis-api.service.hmpps.dsd.io/elite2api/swagger-ui.html#//offenders/getAddressesByOffenderNoUsingGET)
+
```

Comment:
sadly Elite2 is uncredibly inconsistent, so following Elite2 naming conventions isn't possible as they use different names all the time

Reply:
Hi @starswan do you have an example of this? `offenderId` and `bookingId` appear to be used consistently in https://gateway.prod.nomis-api.service.hmpps.dsd.io/elite2api/swagger-ui.html